<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>687</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Case</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Events</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="events_list">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>true</bool>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Physical Evidence</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="physical_evidence_list">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>true</bool>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>319</width>
          <height>416</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Record event</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="start_time_field">
             <property name="date">
              <date>
               <year>2016</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
             <property name="calendarPopup">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="stop_time_field">
             <property name="date">
              <date>
               <year>2016</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="comments_field">
           <property name="placeholderText">
            <string>Comments</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Select device in evidence table</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="add_event_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>333</width>
          <height>416</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Physical evidence</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Time and date seized</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="seized_date_field">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="unique_identifier_field">
           <property name="placeholderText">
            <string>Unique Identifier or serial number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="device_description_field">
           <property name="placeholderText">
            <string>Short device description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="additional_information_field">
           <property name="placeholderText">
            <string>Additional information</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="add_physical_button">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
